docker-compose.dev.yml

```diff
@@ -18,10 +21,8 @@ services:
       - RAILS_ENV=development
       - DOMAIN=app:3000
       - RAILS_SERVE_STATIC_FILES=true
-      - TIMEOUT_MINUTES
-      - TIMEOUT_WARNING_MINUTES
-      - TRAINING_MODULES
-      - BOT_TOKEN
+      # - CONTENTFUL=
+      - CONTENTFUL=y
     volumes:
```

Comment:
Can we stick to `true`

Reply:
So that's a yes? Agreed I don't want to rely on ignore env vars.

Reply:
I meant these lines will be deleted before I mark this PR as ready for review, but I will use `true` for consistency.